core/src/main/scala/com/softwaremill/react/kafka/ReactiveKafka.scala

```diff
@@ -95,15 +96,67 @@ class ReactiveKafka {
     Sink.fromGraph(new KafkaGraphStageSink(producer))
   }
 
-  def graphStageSource[K, V](props: ConsumerProperties[K, V]) = {
+  // Scala DSL
+  def graphStageSource[K, V](
+    props: ConsumerProperties[K, V]
+  ) = {
     val consumer = ReactiveKafkaConsumer(props)
     Source.fromGraph(new KafkaGraphStageSource(consumer))
   }
 
+  // Scala DSL
+  def graphStageSource[K, V](
+    props: ConsumerProperties[K, V],
+    topicsAndPartitions: List[TopicPartition]
+  ) = {
+    val consumer = ReactiveKafkaConsumer(props, topicsAndPartitions)
+    Source.fromGraph(new KafkaGraphStageSource(consumer))
+  }
+
+  // Scala DSL
+  def graphStageSource[K, V](
+    props: ConsumerProperties[K, V],
+    topicsAndPartitions: List[TopicPartition],
+    topicPartitionOffsetsMap: Map[TopicPartition, Long]
```

Comment:
I agree that they should be two functions, but one receives only topicsAndPartitions: List[TopicPartition] because we dont care about the offset, and the other should receive topicsAndPartitionsAndOffsets: Map[TopicPartition, offset]